tests/ReturnTypeTest.php

```diff
@@ -339,6 +339,14 @@ function getBarWithIsset() {
                             return null;
                         }
                     }'
+            ],
+            'resourceReturnType' => [
+                '<?php
+                    namespace bar;
+
+                    function fooFoo() : ?resource {
```

Comment:
`resource` is not a valid return type in PHP 7.1. Use `/** @return ?resource **/` instead.